### PR TITLE
Data #2

### PR DESCRIPTION
**Pull request by jonathastavares**
This is the second milestone for my JavaScript API project, in this milestone, I added submit score and refresh board functions.

Built with:
- HTML
- CSS
- JavaScript
- Webpack

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    userContain.innerHTML = user;
+    const scoreContain = document.createElement('span');
+    scoreContain.innerHTML = score;
+    listItem.appendChild(userContain);
+    listItem.appendChild(scoreContain);
+    listItem.classList.add('d-flex');
+    listItem.classList.add('justify-content-around');
+    list.appendChild(listItem);
+  });
+}
```
Comment (sinansevgi) [inline src/main.js]: - [x] Unless you need to use the `this` keyword in the scope of your function, You should stick with arrow functions. Please refactor this function with the arrow function definition.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                        <!-- Insert Scores here -->
+                    </ul>
+                </div>
+            </div>
+            <div class='right-container d-flex justify-content-center'>
+                <h3 class='add-score'>Add your score</h3>
+                <div class='input-container'>
+                    <input id='name' class='input-box border' type='text' placeholder="Your name">
+                    <br>
+                    <input id='score' class='input-box border' type='text' placeholder="Your score">
```
Comment (sinansevgi) [inline src/index.html]: [Optional]

- [x] It will be better if you have input validation. Currently, I can submit anything as a score.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @jonathastavares, 

Good job so far!
Great use of fetch, async, and await :+1: 
There are some issues that you still need to work on to go further but you are almost there!



 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @jonathastavares , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**